tests/route/test_forced_mgmt_route.py

```diff
@@ -0,0 +1,225 @@
+import ipaddress
+import json
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+from tests.override_config_table.utilities import backup_config, restore_config, \
+        reload_minigraph_with_golden_config
+from tests.syslog.syslog_utils import is_mgmt_vrf_enabled
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.device_type('vs')
+]
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def backup_restore_config(duthosts, enum_rand_one_per_hwsku_hostname):
+    """make sure tacacs server running after UT finish"""
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    #  Backup config before test
+    CONFIG_DB = "/etc/sonic/config_db.json"
+    CONFIG_DB_BACKUP = "/etc/sonic/config_db.json_before_override"
+    backup_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
+
+    yield
+
+    #  Restore config after test finish
+    restore_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
+
+
+def get_interface_reload_timestamp(duthost):
+    res = duthost.command("sudo service interfaces-config status "
+                          "| grep 'Finished interfaces-config.service - Update interfaces configuration.'")
+    logger.info("interfaces config timestamp {}".format(res["stdout_lines"]))
+
+    if len(res["stdout_lines"]) == 0:
+        return ""
+
+    return res["stdout_lines"][-1]
```

Comment:
>res["stdout_lines"][-1] [](http://example.com/codeflow?start=11&length=23)

If the output change a little bit, like a fixed non-time string at the end, will it fail or blindly succeed?

Reply:
Fixed, change to read ExecMainExitTimestamp from stdout.